jobs/env/ci-kubernetes-e2e-gci-gce-latest-downgrade-kube-proxy-ds.env

```diff
@@ -2,3 +2,13 @@
 
 STORAGE_MEDIA_TYPE=application/vnd.kubernetes.protobuf
 KUBE_PROXY_DAEMONSET=false
+
+# TODO: This is pinned to the 1.8 version of etcd. 1.9 was changed to
```

Comment:
nit: `TODO(MrHohn)`?

Reply:
In fact, I copied this from https://github.com/kubernetes/test-infra/blob/master/jobs/env/ci-kubernetes-e2e-gce-new-master-upgrade-cluster.env, not sure who owns this.

Or maybe I can just remove the TODO here as we don't really care about which etcd version to use :)

Reply:
feel free to ignore my bikesheds :see_no_evil: :hear_no_evil: :speak_no_evil:

Reply:
We shouldn't have TODO(name) in the codebase...